libr/core/panels.c

```diff
@@ -37,30 +37,30 @@
 
 static RConsCanvas *can;
 
+typedef enum {
+	PANEL_TYPE_FRAME = 0,
+	PANEL_TYPE_MENU = 1
+} panel_type;
```

Comment:
Typedefs in r2land are camelcase

Reply:
ive seen typedef called inflate_mode. but okay then i would fix like that. i love camel case anyway

Reply:
you sure about camel case though cuz i searched very roughly now and already found bunch of hoge_hoge kinda manners

Reply:
done

Reply:
inflate_mode is from a shlr so it's just external libs.